src/Microsoft.AspNet.Mvc.ViewFeatures/Rendering/IHtmlHelper.cs

```diff
@@ -133,7 +135,7 @@ public interface IHtmlHelper
             string controllerName,
             object routeValues,
             FormMethod method,
-            bool suppressAntiforgery,
+            bool? suppressAntiforgery,
```

Comment:
@dougbu, What is your opinion on adding another overload for `BeginForm` and `BeginRouteForm` on `IHtmlHelper`?